tests/v2/test_stack_aggregator.py

```diff
@@ -41,6 +41,37 @@ def _request_body():
     }
 
 
+def _go_request_body():
```

Comment:
Define separate test file for golang

Reply:
https://github.com/fabric8-analytics/f8a-server-backbone/pull/252#discussion_r526802915